### PR TITLE
SAK-41826 Roster: Surface user pronunciation

### PR DESCRIPTION
**Pull request by mpellicer**

### CODE REVIEW

File: roster2/tool/src/java/org/sakaiproject/roster/impl/SakaiProxyImpl.java [hunk 1]
```diff
@@ -525,6 +532,45 @@ public RosterMember getMember(String siteId, String userId, String enrollmentSet
             } catch (URISyntaxException e) {
                 log.error("URI Syntax Error", e);
             }
+        } else{
```
[the remark this answers is not in the document]
Comment (mpellicer) [reply roster2/tool/src/java/org/sakaiproject/roster/impl/SakaiProxyImpl.java]: outdated comment? I did it!

File: roster2/tool/src/java/org/sakaiproject/roster/impl/SakaiProxyImpl.java [hunk 2]
```diff
@@ -1281,6 +1338,22 @@ public boolean getShowVisits() {
         return serverConfigurationService.getBoolean("roster.showVisits", false);
     }
 
+    /**
+     * {@inheritDoc}
+     */
+    public Boolean getViewUserNamePronunciation() {
+        if(this.isNamePronunciationEnabled()) {
```
[the remark this answers is not in the document]
Comment (mpellicer) [reply roster2/tool/src/java/org/sakaiproject/roster/impl/SakaiProxyImpl.java]: outdated comment? I did it!

File: roster2/tool/src/webapp/js/roster.js [hunk 3]
```diff
@@ -543,6 +545,36 @@
     if (!renderAll) {
         $(window).trigger('scroll.roster.rendered');
     }
+
+    //Associate the members audio with start and stop actions
+    var $allAudioElem = $('.audioPlayer');
+    $.each(members, function(index, member){
+        var audioId = '#audio-' + member.userId;
```
[the remark this answers is not in the document]
Comment (mpellicer) [reply roster2/tool/src/webapp/js/roster.js]: Did the first part.

**Comment by mpellicer** [comment]
Thank you for the review @adrianfish, ready to go?

**Comment by adrianfish** [comment]
Go for it. But if you could find it in your heart to move to 2 spaces for tabs, I would be filled with joy :) If you haven't got time, don't worry about it, just merge.

**Comment by mpellicer** [comment]
Done!

**Pull request merged**